src/syslog_udp_sender.erl

```diff
@@ -0,0 +1,129 @@
+%%%-------------------------------------------------------------------
+%%% @author tihon
+%%% @copyright (C) 2016, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 17. Jun 2016 15:18
+%%%-------------------------------------------------------------------
```

Comment:
The headers of the project reflect the software license of the respective file (some files use MIT, some must be Apache 2.0).

I would suggest adding the MIT-specific header with your name in the copyright section.